torch/testing/_internal/common_fsdp.py

```diff
@@ -629,17 +650,24 @@ def forward(self, x):
     def get_loss(self, input, output):
         loss = self.module.get_loss(input, output)
         if self.delay_after_loss_ms > 0:
-            torch.cuda._sleep(int(self.delay_after_loss_ms * get_cycles_per_ms()))
+            if TEST_HPU:
+                time.sleep(self.delay_before_reduction_ms / 1000)
+            elif TEST_CUDA:
+                torch.cuda._sleep(int(self.delay_after_loss_ms * get_cycles_per_ms()))
```

Comment:
why is HPU sleeping based on `delay_before_reduction_ms` instead of `delay_after_loss_ms`?

Reply:
this one I just added is new